app/containers/CheckIcon/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { RadioButton } from 'react-native-ui-lib';
+
+import { useTheme } from '../../theme';
+
+export const CheckRadioButton = ({
```

Comment:
I'm not sure about this function and file name.
Checkbox and Radio button are different things https://material.io/components/radio-buttons#usage

I think it's worth keeping it as RadioButton, because of size and colors.